### DISCUSSION TITLE
The university does not provide education email addresses to students, but I still need to complete the verification.

### DISCUSSION THREAD

**Discussion by xmgzxmgz** [community/community#136903] [GitHub Education]
### Hi everyone,

Dear GitHub Education Team,

I am a student at Fuzhou University of International Studies and Trade. Although our university has established an education email system, it only provides edu domain emails to faculty members and not to students. I have repeatedly uploaded my student ID, admission letter, and verification documents from the China Higher Education Student Information Network (https://www.chsi.com.cn/), but I have still been unable to complete the GitHub Education verification.

I am more than willing to provide any additional materials for verification. Your assistance in this matter would be greatly appreciated.

Thank you very much for your help.

Sincerely,
Xmgz

**Comment by xmgzxmgz** [comment]
The email response indicated: 'Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page.'

**Comment by samus-aran** [comment]
Hi @xmgzxmgz  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
